### PR TITLE
Add default dimension.

### PR DESCRIPTION
**Pull request by wangfakang**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, 
details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/filter/stream/faulttolerance/regulator/measure_model.go [hunk 1]
```diff
@@ -91,7 +91,8 @@ func (m *MeasureModel) snapshotInvocations(recoverTime int64) []*InvocationStat
 			return true
 		}
 		if stat.GetCall() <= 0 {
-			if stat.AddUselessCycle() {
+			// don't release unhealth stat host
+			if stat.AddUselessCycle() && stat.IsHealthy() {
```
Comment (NeGnail) [inline pkg/filter/stream/faulttolerance/regulator/measure_model.go:95]: there are already unhealthy judgments in front of this cycle. And with one, the semantics are more complete.
